core/src/cluster_info.rs

```diff
@@ -197,6 +211,12 @@ enum Protocol {
     PruneMessage(Pubkey, PruneData),
 }
 
+struct ResponseScore {
```

Comment:
Sorry if I'm sounding pedantic, how about details about what the x and y axis are?